content-src/components/Snippet/Snippet.scss

```diff
@@ -0,0 +1,77 @@
+.snippet {
+  $snippet-padding: 26px;
+  $snippet-bg-color: rgba($white, 0.95);
+  $snippet-divider-color: rgba($black, 0.1);
+  $snippet-width: 500px;
+  $snippet-base-font-size: 12px;
+  $snippet-title-size: 14px;
```

Comment:
I'd be happy to have a broader discussion about `rem`s – I personally think it's OK, maybe even better to use pixels – but for this case, probably best to stick with what the rest of the project uses (i.e. pixels)